interface-definitions/include/vrrp/garp.xml.i

```diff
@@ -0,0 +1,74 @@
+<!-- include start from vrrp/garp.xml.i -->
+<node name="garp">
+  <properties>
+    <help>Gratuitous ARP parameters</help>
+  </properties>
+  <children>
+    <leafNode name="master-delay">
+      <properties>
```

Comment:
There is no need to write default to the help string. This is done automagically by the node.def geberator based on the defaultValue.

Reply:
Ok, I'll remove all "default" descriptions